pyrt/light/pointlight.py

```diff
@@ -12,15 +12,16 @@ class PointLight(Light):
 
     """Class describing a point light source"""
 
-    def __init__(self, position: Vec3) -> None:
+    def __init__(self, position: Vec3, coef: float) -> None:
         """
         Constructor
 
         Not yet complete - params will change in near future!
         :param position: position of the light
         """
-        Light.__init__(self, "PointLight")
+        Light.__init__(self, coef, "PointLight")
```

Comment:
You should not touch old one code if it is really not necessary to. So please delete the **coef** param from point-light

Reply:
This is the intensity of the light source as a parameter, without this coefficient the picture is not so beautiful, but now it is deleted everywhere